bin/etl.js

```diff
@@ -1,130 +1,120 @@
 #!/usr/bin/env node
-'use strict';
-const { Client } = require('pg');
-const copyFrom = require('pg-copy-streams').from;
-const http = require('http');
-const path = require("path");
+/* eslint-disable no-console */
+
 const fs = require('fs');
+const request = require('request');
 const unzip = require('unzip');
-const copyTo = require('pg-copy-streams').to;
-const { exec } = require('child_process');
 const iconv = require('iconv-lite');
+const copyFrom = require('pg-copy-streams').from;
+const { Client } = require('pg');
 
+const downloadFile = (url, fileName) =>
+  new Promise((resolve, reject) => {
+    console.log('Downloading file');
+    request(url)
+      .on('error', (err) => {
+        console.log(err);
+        reject(err);
+      })
+        .pipe(fs.createWriteStream(fileName))
+        .on('close', () => {
+          resolve('success');
+        });
+  });
 
-// Ensure environment variables are read.
-require('../config/env');
+const unzipFile = (pathToZip, pathToExtract) =>
+ new Promise((resolve, reject) => {
+   console.log('Extracting file from zip');
+   fs.createReadStream(pathToZip)
+      .pipe(unzip.Extract({ path: pathToExtract }))
+      .on('error', (error) => {
+        console.log(error);
+        reject(error);
+      })
+      .on('close', () => {
+        resolve();
+      });
+ });
 
-if (process.env.NODE_ENV !== 'production') {
-  let dotenv = require('dotenv');
-  dotenv.config();
-}
+const truncateTable = async () => {
```

Comment:
This does not need the `async` prefix nor the `await` in the return  since you are returning a promise that you are awaiting on the invocation of this function. [Here](https://jakearchibald.com/2017/await-vs-return-vs-return-await/) is an article describing the only use case I can think of for returning an awaited function.

Reply:
Alright, I'll fix these errors. Thanks, for the article!